website_chd_product_configurator/static/src/js/website_chd_product_configurator.js

```diff
@@ -0,0 +1,103 @@
+$(document).ready(function () {
+	console.log(openerp);
+	$('.type_selection').each(function () {
```

Comment:
you don't need to loop through the elements to attach event handlers.
`jQuery('.type_selection').on(...)` does the same